srcpkgs/oneko/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'oneko'
+pkgname=oneko
+version=1.2.5
+revision=1
+# other archs end with /usr/bin/xmkmf: line 53: 15231 Aborted
+archs="x86_64"
+_pkgver=1.2.sakura.5
```

Comment:
why not just set `version=1.2.sakura.5`? alphabetic characters are allowed in versions

Reply:
didn't know that. thanks